packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+            closeButton={false}
+            modalSize='md'
+        >
+            {viewType === 'transfer' &&
+            <StyledContainer className='small-centered'>
+                <img className='transfer-img' src={nft.metadata.mediaUrl} alt='NFT'/>
+
+                <h3><Translate id='NFTTransfer.transferNft'/></h3>
+                <p className='transfer-txt'><Translate id='NFTTransfer.enterReceipt'/></p>
+
+                <form>
+                    <div className='receiver-input'>
+                        <ReceiverInputWithLabel
+                            receiverId={receiverId}
+                            handleChangeReceiverId={receiverId => setReceiverId(receiverId)}
+                            checkAccountAvailable={accountId => dispatch(checkAccountAvailable(accountId))}
+                            localAlert={localAlert}
+                            autoFocus={!isMobile()}
+                            clearLocalAlert={() => dispatch(clearLocalAlert())}
+                            setAccountIdIsValid={setAccountIdIsValid}
+                        />
+                    </div>
+
+                    <ModalFooter>
+                        <div className='buttons'>
+                            <FormButton
+                                className='link'
+                                type='button'
+                                onClick={onClose}
+                                color='gray'
+                            >
+                            <Translate id='NFTTransfer.cancel'/>
+                            </FormButton>
+                            <FormButton
+                                className='next-btn'
+                                type='submit'
+                                disabled={!accountIdIsValid}
+                                onClick={() => setViewType('confirm')}
+                            >
+                            <Translate id='NFTTransfer.next'/>
+                            </FormButton>
+                        </div>
+                    </ModalFooter>
+                </form>
+            </StyledContainer>
+            }
+
+            {viewType === 'confirm' &&
+                <StyledContainer className='small-centered'>
+                    <h3><Translate id='NFTTransfer.transferNft'/></h3>
+
+                    <div className='confirm-nft-card'>
+                        <div className='confirm-img'>
+                            <img src={nft.metadata.mediaUrl} alt='NFT'/>
+                        </div>
+
+                        <div className='line'></div>
+                        <div className='from-box'>
+                            <span className='confirm-txt v-center'><Translate id='transfer.from'/></span>
+                            <span className='h-right v-center'>
+                                <span className='account-id'>{accountId}</span>
+                                <div className='amount-grey'>{balanceToShow} NEAR</div>
```

Comment:
Please use the[ `<Balance/>`](https://github.com/near/near-wallet/blob/master/packages/frontend/src/components/wallet/Wallet.js#L365) component wherever we show token amounts

Reply:
sure, will do